cdap-docs/application-templates/source/etl/templates/sinks/batch/cube.rst

```diff
@@ -42,9 +46,78 @@ cubeFact.timestamp.field is provided).
 **cubeFact.measurements:** Measurements to be extracted from StructuredRecord to be used in CubeFact.
 Provide properties as a JSON Map. For example, to use the 'price' field as a measurement of type gauge,
 and the 'count' field as a measurement of type counter, the property should have the value:
-{"cubeFact.measurement.price":"GAUGE", "cubeFact.measurement.count":"COUNTER"}.
+{"cubeFact.measurement.price":"GAUGE", "cubeFact.measurement.quantity":"COUNTER"}.
 
 .. rubric:: Example
 
-TODO: Fill me out
-
+The following configuration tells to write data into Cube dataset with name "myCube"; provides dataset properties
+(dataset.*) for creating new dataset, if the one with given name doesn't exist; configures measurements to aggregate and
+how to determine timestamp for the facts to be written into Cube::
```

Comment:
This configuration specifies writing data into a Cube dataset named "myCube"; it provides dataset properties
(dataset.*) for creating new a dataset, if one with the given name doesn't exist; it then configures measurements to aggregate and specifies the timestamp format for facts being written into the Cube::